.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
+ignore = E203, E266, E501, W503, F403, F401
+max-line-length = 79
+max-complexity = 18
+select = B,C,E,F,W,T4,B9
```

Comment:
Is it possible to create a brief markdown file of what coding standards we are following in this project or a link to them? 

Something similar to this:
https://github.com/ColoredCow/portal/blob/master/docs/coding-standards.md